src/alarms/notifications/raster_alarms/RasterAlarmForm.tsx

```diff
@@ -209,6 +209,7 @@ const RasterAlarmForm: React.FC<Props & DispatchProps & RouteComponentProps> = (
           errorMessage={required('Please select a raster', values.raster)}
           triedToSubmit={triedToSubmit}
           isAsync
+          isCached
```

Comment:
Added this optional parameter to the SelectDropdown to specify if cache option is activated for the async select dropdown. The SelectDropdown for asset selection which cannot be cached is the reason I added this parameter.

Reply:
Probably good. I do not yet understand why Asset selection cannot be cached.
Could you explain a bit?

Reply:
Yes, so basically, the search endpoint for assets includes a type parameter, this type parameter can be changed using the asset type dropdown. If the async search is cached then even if the asset type is changed, previous search is still cached and will give us outdated list of assets.